### ISSUE TITLE
SizedBox for whitespace warning

### ISSUE THREAD

**Issue by JoseAndresMejiasRojas** [passsy/dart-lint#19]
Hello, guys. I just update lint 1.3.0. in one of my projects.

Then, I got a warning with this code:

```dart
@override
Widget build(BuildContext context) {
   return Container(width: 270, child: _video);
}
```
The warnings says `SizedBox for whitespace`.

![image](https://user-images.githubusercontent.com/38768936/94736590-b7600d80-0329-11eb-8a81-b7e35b83fc1f.png)

I think it's a false positive. What do you think?

**Comment by passsy** [comment]
No, your `Container` can be replaced with `SizedBox`.

```dart
@override
Widget build(BuildContext context) {
   return SizedBox(width: 270, child: _video);
}
```

**Comment by JoseAndresMejiasRojas** [comment]
Indeed you're right. My bad, I forgot that a SizedBox can have a child widget. Thanks!
